### PR TITLE
add: Sns integration for the Defilama Governance

### PR DESCRIPTION
**Pull request by NikolasHai**
This MR adds the integration of Service Nervous Systems to the Defilama Governance repo

### CODE REVIEW

File: defi/package-lock.json
Comment (g1nt0ki) [inline defi/package-lock.json:1]: can you revert change to this file, we are bit paranoid and like doing all package related changes ourselves

File: defi/src/governance/compound.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { updateStats, toHex, getGovernanceSources, getChainNameFromId } from './
 import * as sdk from '@defillama/sdk'
 import { sliceIntoChunks } from '@defillama/sdk/build/util/index'
 import { getProvider } from '@defillama/sdk/build/general'
-import { addICPProposals } from './icp'
+import { addICPProposals } from './nns'
```
Comment (g1nt0ki) [inline defi/src/governance/compound.ts:8]: this file doesnt exist, is the PR complete? maybe you forgot to commit nns.ts?

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: defi/src/governance/compound.ts (hunk 1 above)
Comment (NikolasHai) [reply defi/src/governance/compound.ts:8]: Yes thanks for pointing that out, I renamed the file and forgot to add it afterwards.

**Comment by g1nt0ki** [comment]
@NikolasHai thanks for the PR, I have made some minor fixes but for some reason, was unable to push here. So, I have merged it here: https://github.com/DefiLlama/defillama-server/pull/4729

**Pull request closed without merging**